manage.py

```diff
@@ -113,4 +113,5 @@ def parse_args():
 
     from django.core.management import execute_from_command_line
 
-    execute_from_command_line([sys.argv[0]] + django_args)
+    sys.argv[1:] = django_args
```

Comment:
This change looks like something is accessing sys.argv directly? I'm just curious what it is.